app/components/allocation_component.html.erb

```diff
@@ -11,5 +15,9 @@
     <%= render GovukComponent::WarningTextComponent.new(text: 'These allocations are for the current winter 2021 Get help with tech programme. Devices ordered before 30 September 2021 are not included here.', classes: 'govuk-!-margin-bottom-0') %>
   </div>
 
+  <div id="current_tranche">
+    <%= render GovukComponent::WarningTextComponent.new(text: "The 'Devices ordered' and 'remaining devices' may not add up to the 'Total allocation' shown if you are a managed school or there are other temporary restrictions on ordering." , classes: 'govuk-!-margin-bottom-0') %>
```

Comment:
Might need to use the fancy curly single quotes!

Reply:
Changed!